tests/python/pants_test/pantsd/test_pailgun_server.py

```diff
@@ -7,15 +7,22 @@
 import socket
 import threading
 import unittest
+from builtins import bytes
 from contextlib import contextmanager
-from SocketServer import TCPServer
+from socketserver import TCPServer
 
 import mock
+from future import standard_library
 
 from pants.java.nailgun_protocol import ChunkType, NailgunProtocol
 from pants.pantsd.pailgun_server import PailgunHandler, PailgunServer
 
 
+standard_library.install_aliases()
+
+
+
```

Comment:
I expect that CI will fail on this.

Recommend either manually running `build-support/bin/pre-commit.sh`, or installing (and skipping when necessary) the commithooks, as described here: https://www.pantsbuild.org/howto_contribute.html#getting-pants-source-code

Reply:
You're right Stu. We don't want to ever use this idiom because isort changes the order where it no longer works as expected. `standard_library.install_aliases()` is supposed to happen *before* the relevant import, but isort always moves it below every import.

Instead, we have to use `from future.moves import x`. The tricky part is figuring out what `standard_library.install_aliases()` was overriding. I use autocomplete and try out every import to see where it's coming from, e.g. see if `future.moves.contextlib` exists. Sometimes I can't find anything, so I delete the import because I think it might be an over-optimization?

Reply:
I'm also not sure why this line was being added. None of the above libraries should need `future.moves`. I think you can delete this + `from future import standard_library`

Reply:
IIRC, you can also use `# noqa` comments with isort? Not certain though.

Reply:
But the reason I mentioned that this will fail is that there is a style check for 2 blank lines between top level definitions.

Reply:
Unfortunately `# noqa` does not work with isort :/ The first day of this project was me spending 4-5 hours trying to hack a solution to getting isort to stop moving the line, until finally discovering `future.moves`